app/assets/javascripts/src/fileUpload.js

```diff
@@ -18,11 +18,54 @@
 
         $element.change(function(){
             $('#error-summary').remove();
-            // $('#message-warning').remove();
             $('#saveAndContinueButton').attr('disabled','disabled');
 
             $(this).after('<div class="message-warning" id="message-warning"><p>Please wait whilst your file is uploading. This may take some time.</p></div>');
+
             var file = this.files[0];
+            function resolveMimeType(upload) {
+                if(file.type != "" && file.type != undefined){
+                    return file.type;
+                }
+                var extension = upload.name.substr( (upload.name.lastIndexOf('.') +1) );
+                var mime;
+                switch(extension){
+                            case "xls":
+                            mime = "application/vnd.ms-excel";
+                            break;
+
+                            case "xlsb":
+                            mime = "application/vnd.ms-excel.sheet.binary.macroEnabled.12";
+                            break;
+
+                            case "xlsx":
+                            mime = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet";
+                            break;
+
+                            case "pdf":
+                            mime = "application/pdf";
+                            break;
+
+                            case "docx":
+                            mime = "application/vnd.openxmlformats-officedocument.wordprocessingml.document";
+                            break;
+
+                            case "doc":
+                            mime = "application/msword";
+                            break;
+
+                            case "jpg":
+                            mime = "image/jpeg";
+                            break;
+
```

Comment:
add png. image/png

Reply:
I have included PNG and also messages file updated Allowed File Types